modules/extensions-support/src/test/java/org/mule/module/extension/Ricin.java

```diff
@@ -42,4 +48,10 @@ public int hashCode()
     {
         return microgramsPerKilo.hashCode();
     }
+
+    @Override
+    public String kill()
+    {
+        return "You have been killed with Ricin";
```

Comment:
you should use the RICIN_KILL_MESSAGE constant here. It also looks like that constant should be moved to this class